### PR TITLE
Smashing magazine feature branch

### PR DESCRIPTION
**Pull request by mosesmuchemi**
Creating a pull request for automated stickler review and finally a manual review by a TSE

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+            </article>
+        </div>
+        <div class="container">
+            <a href="" class="block w-100 light sm-height">Lorem ipsum dolor sit amet.</a>
+        </div>
+    </main>
+
+    <!--     FOOTER         -->
+    <footer class="dark big-height w-100 padding-100">
+        <h4>Footer Section</h4>
```
Comment (bolah2009) [inline index.html]: Let's wrap the footer content with anchor tags `<a>`.

 - [x]  It contains centered links (content is not important).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+    <title>Smashing Mag Grayscale</title>
+</head>
+<body>
+
+    <!--         HEADER          -->
+    <header class="dark">
+        <nav>
+            <!-- <div class="logo">Logo</div> -->
+            <p>Header Section</p>
+        </nav>
```
Comment (bolah2009) [inline index.html:38]: Let's add the navigation element with `div`

##### Navigation bar

* [x] It uses `div` and positions elements as at the source page (content is not important).

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Good job team :clap: 

### [The clone](https://rawcdn.githack.com/mosesmuchemi/anibaba-muchemi-smashing-magazine-grayscale/7d1c0c6a4f20c1ce1d08ad7bc8693286de8f976c/index.html) looks good. :blush: 

Please let's make the suggested changes in the comments below.


#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

File: index.html (hunk 1 above)
Comment (mosesmuchemi) [reply index.html]: implemented by @silverside3000

File: index.html (hunk 2 above)
Comment (mosesmuchemi) [reply index.html:38]: implemented by @silverside3000

**Review by stratospherique — APPROVED** [review]
### Nice work implementing the suggested changes :clap: . Your project is approved :heavy_check_mark:  and you may merge to master :+1: :+1:

**Pull request merged**